backend/requirements.in

```diff
@@ -1,3 +1,3 @@
-tfx==1.2.0
```

Comment:
this is causing all tests to fail, because we need tfx when compiling samples

Reply:
Thank you Yuan! I added back TFX (tfx 1.4 supports ml-metadata 1.4) and resolved a few dependencies requirements.

The new change has presubmit failure during APIserver build, looks like it is failing because it cannot find `output_dataset_two` in the two steps sample pipeline: https://pantheon.corp.google.com/cloud-build/builds/35ddfc6b-a057-4603-98ef-1e74ebb79f5b;step=7?project=ml-pipeline-test.

Is it because we recently re-enabled V2 mode tests?

Reply:
@zijianjoy It's because we have a v2 compatible sample pipeline in https://github.com/kubeflow/pipelines/blob/8aee62142aa13ae42b2dd18257d7e034861b7e5e/backend/src/apiserver/config/sample_config.json#L22-L27, but we removed v2 compatible support in latest SDK.

cc @chensun
Looks like we need to remove that sample as well.

Reply:
I tried to fix this for you, because I also need the upgraded MLMD library.

Reply:
Thank you Yuan for fixing it! The memory limit is added to xgboost pipeline, instead of `[Tutorial] V2 lightweight Python components` which is the failure I saw in this thread. Is it expected? How did we fix the latest SDK not having V2 compatible support issue?